test/steps/setup.js

```diff
@@ -79,15 +79,21 @@ const setUpReadlineStubs = () => {
 };
 
 function setUpLiskJSAPIStubs() {
+	const sendRequestDefaultResult = { success: true };
```

Comment:
Is this needed here or will we always define it later?

Reply:
it's the default result, if we don't need to change it, we can use as it is.